README_py.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to a Toy PyBind11 Algorithm
```

Comment:
We should think about the best place to put a readme like this.
It will obviously be dictated by the content and how we want to strcture the flow, but it would be nice to start thinking about some flow/file organization that we think will help people the best.